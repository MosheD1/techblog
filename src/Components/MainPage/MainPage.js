import React from 'react';
import BlogList from '../BlogList/BlogList';

import './MainPage.css'

const MainPage = ({blogs , name, onSubRouteChange}) => {
    return(
        <div>
            <h1 className='title'>{`Wellcome ${name}, let's create new content`}</h1>
            <div className='button'>
                <button onClick={() => onSubRouteChange('newBlog')}>New Blog</button>
            </div>
            <BlogList blogs={blogs}/>
        </div>
    )
}

export default MainPage;