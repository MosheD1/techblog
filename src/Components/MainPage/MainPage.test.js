import React from 'react';
import MainPage from './MainPage';
import {shallow, render, mount} from 'enzyme';

describe('MainPage component', () => {
    it('expect to render MainPage', ()=> {
        expect(shallow(<MainPage />)).toMatchSnapshot();
    });
});