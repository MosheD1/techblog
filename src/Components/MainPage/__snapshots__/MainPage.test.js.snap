// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component expect to render MainPage 1`] = `ShallowWrapper {}`;
